src/wasm32/ffi.c

```diff
@@ -32,37 +32,9 @@
 
 #include <emscripten/emscripten.h>
 
-#ifdef DEBUG_F
```

Comment:
I'd vote to keep this personally.

Reply:
I wasn't sure if it was necessary, but it may indeed be useful. Added back with the latest revision.